src/index.ts

```diff
@@ -45,9 +45,39 @@ function notUndefined<T>(x: T | undefined): x is T {
   return x !== undefined;
 }
 
-export async function dump(n: readonly SceneNode[]): Promise<F.DumpedFigma> {
+// Returns true if n is a visible SceneNode or
+// not a SceneNode (e.g. a string, number, etc.)
+function isVisible(n: any) {
+  if (typeof n !== "object") {
+    return true;
+  }
+
+  if (
+    !("visible" in n) ||
+    typeof n.visible !== "boolean" ||
+    !("opacity" in n) ||
+    typeof n.opacity !== "number" ||
+    !("removed" in n) ||
+    typeof n.removed !== "boolean"
+  ) {
+    return true;
+  }
+
+  return n.visible && n.opacity > 0 && !n.removed;
+}
+
+export async function dump(
+  n: readonly SceneNode[],
+  skipInvisibleNodes: boolean = true
```

Comment:
I think in the future, this will want to be an options object rather than a single param. Since we're in the 0.x version, breaking changes are ok, but will be easier to maintain API compatibility with an object.

Reply:
Good idea.

Reply:
Done.